eng/build.ps1

```diff
@@ -262,6 +262,10 @@ if ($vs) {
   # Disable .NET runtime signature validation errors which errors for local builds
   $env:VSDebugger_ValidateDotnetDebugLibSignatures=0;
 
+  # MSBuild Server dogfood.
+  $env:DOTNET_CLI_USE_MSBUILD_SERVER=1;
```

Comment:
Do we really want to set it without a way to opt out? What if we hit a problem with the server that would prevent local builds from succeeding and all developers would get affected?

Reply:
That is not as simple when you often build different branches, which I do during development. Editing a file for each branch, paying attention to not to accidentally add the change to your change list etc. sounds like a pain. 
The way to opt out can be as simple as not setting the DOTNET_CLI_USE_MSBUILD_SERVER if it was already set. Then in case there are some issues, I can set that variable to 0 until the issue is resolved. I can have that set globally so all shells I run would pick it.